definitions/tools/multiqc.cwl

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: "multiqc"
+baseCommand: ["/usr/local/bin/multiqc"]
+requirements:
+    - class: ShellCommandRequirement
+    - class: ResourceRequirement
+      ramMin: 4000  
+    - class: DockerRequirement
+      dockerPull: "mgibio/multiqc:1.9"
+    - class: StepInputExpressionRequirement
+
+arguments: [
+    "--zip-data-dir",
+    "-n", "multiqc_out"
+    
+]
+ 
+inputs:
+  inputfiles_array:
```

Comment:
Is there a concise description of what sort of inputs these are that we could use for the name? The line below this one indicates that it's an array of files and the line above indicates it's an input, so it might be nice to have an identifier that is more specific about what goes here.  (A `doc` line could also be useful.)

Reply:
Ye this was something i was confused too, what to add?!  Multiqc supports log files from 96 different tools, https://multiqc.info/#supported-tools may be just add the weblink?

Reply:
Yeah, a link in the docs could be useful!  Maybe the input name could be something like `qc_sources` or `source_qc_tool_results` or something like that?